zulip_bots/zulip_bots/run.py

```diff
@@ -19,7 +19,9 @@ def import_module_from_source(path, name=None):
     if not name:
         name = splitext(basename(path))[0]
 
-    if six.PY2:
+    # importlib.util.module_from_spec is supported from Python3.5
+    py_version = sys.version_info
+    if py_version.major < 3 or (py_version.major == 3 and py_version.minor < 5):
         import imp
```

Comment:
Ah, nice catch! I wrote the original code and I missed this. Thanks for taking care of this! Looks good to me. :)

Reply:
One little piece of feedback. I think the commit message for this change could be more informative. `zulip_bots: Fix run.py import for Python < 3.5.` does not seem specific enough. I think a better message would be something along the lines of `zulip_bots: Import imp.load_source for Python < 3.5.`. It would be great if you could please amend the commit message. :)

Reply:
Also, I think we can get rid of line 13 `import six` now that we don't use it.